docs/001_develop/03_client-capabilities/023_custom-components/01_quick-start

```diff
@@ -0,0 +1,82 @@
+# Quick Start
+
+Let's create a simple web component using Genesis. There are three parts that make up a component: the HTML template, the CSS styles and the component logic. Web components can vary in complexity, from a simple button to a very detailed interactive experience.
+
+## Create Custom Element
+
+Start by importing all necessary part:
+
+```shell
+import { css, customElement, GenesisElement, html } from '@genesislcap/web-core';
+```
+
+```shell
+
+# Create an HTML template using the html tag template literal
+const template = html`<button>Click Me</button>`;
+
+# Create CSS styles using the css tag template literal
+const styles = css`
+    h1 {
+      color: grey;
+    }
+`;
+
+# The @customElement decorator is a function that helps define and register a custom HTML element. It associates a class with a specific custom tag name in the DOM.
+
+@customElement({
+  name: 'my-button',
+  template,
+  styles,
+})
+
+export class HelloWorld extends GenesisElement {}
+
+```
+
+We’ve created a custom component called <my-button>, which currently displays a basic button in the browser with the text "Click Me." However, to make it more useful, let's add an attribute that allows us to customize the button's label and give it more dynamic functionality.
```

Comment:
Would wrap `<my-button>` in backticks
Second sentence, remove 'However', => To make it more useful, let's add an attribute that allows us to customize the button's label and give it more dynamic functionality.